activesupport/lib/active_support/core_ext/numeric/round.rb

```diff
@@ -0,0 +1,48 @@
+# <tt>half</tt> option was added to Integer#round and Float#round in Ruby 2.4.
+# And a behavior of round was changed to round to even.
+# We have depended a behavior of rounding to nearest.
+# In some case (i.e. <tt>ActionView::Helpers::DateHelper#distance_of_time_in_words</tt>),
+# we have to pass <tt>half: :up</tt> to <tt>round</tt>.
+#
+# This core_ext enables to us to pass a option,
+# even before Ruby 2.3.
+if !(0.0.round(0, half: :up) rescue false)
+  require "bigdecimal"
+
+  module ActiveSupport::FloatRoundWithOption
+    def round(ndigits = 0, half: :even)
```

Comment:
Them changing the default behaviour from 2.3 to 2.4 seems weird - may break a bunch of finance-related Rails apps :smile: